muxrpc/test/nodejs/sbot_client.js

```diff
@@ -13,8 +13,6 @@ if (testSHSappKey !== false) {
 
 const createSbot = theStack({caps: {shs: testAppkey } })
   .use(require('ssb-db'))
```

Comment:
Just a tip: it should be safe to change this to `ssb-db2` :) , cc @arj03

Reply:
I'm not even sure any of them publish anything to feeds at this point. :sweat_smile: 

I just copied this as _the classic way to make an ssb-server without ssb-server_.

Reply:
Nothing writes to the db, but I think ssb-conn reads from the db (e.g. to know who are friends so that they get automatically connected in the scheduler). It doesn't make any difference to your tests, whether ssb-db or ssb-db2, it just feels nicer to default to ssb-db2

Reply:
Total unbiased opinion here ;-) but default to ssb-db2 will be especially good if we start chasing bugs later in the process.

Reply:
_db2, db2, db2!_

Reply:
@cryptix Actually, I just realized that ssb-conn doesn't strictly require ssb-db2, so you could actually not include neither ssb-db nor ssb-db2, just ssb-conn and ssb-room-client

Reply:
using db2 and db2/compat/db now (because my tests use whomai :upside_down_face: )